pages/live/main.js

```diff
@@ -1,3 +1,6 @@
+// Store the currently relevant list of lobbies globally
+var lobbies;
```

Comment:
Is there any reason you don't use `let` here, given that it's top-level? I know it doesn't matter, but `let` is generally considered best practice (this is super nitpicky)

Reply:
It's a matter of code style, I suppose. I use `var` when I want variables to be explicitly global.